scripts/setup/install_azcopy.sh

```diff
@@ -0,0 +1,7 @@
+echo "Downloadng azcopy ...."
+wget https://aka.ms/downloadazcopy-v10-linux 
+tar -xvf downloadazcopy-v10-linux
+sudo rm /usr/bin/azcopy
+sudo cp ./azcopy_linux_amd64_*/azcopy /usr/bin/
+alias azcopy=/usr/bin/azcopy
+echo "Installed azcopy ...."
```

Comment:
Please move this script one directory up. The scripts/setup directory focuses on Azure resources.

Reply:
Done